.luacheckrc

```diff
@@ -19416,6 +19416,7 @@ globals = {
 	"WOW_MOUSE_NOT_FOUND",
 	"WOW_PROJECT_CLASSIC",
 	"WOW_PROJECT_BURNING_CRUSADE_CLASSIC",
+	"WOW_PROJECT_BURNING_CRUSADE_CLASSIC",
```

Comment:
@cwilson21 @evil-morfar This has to be "WOW_PROJECT_WRATH_CLASSIC" or am i wrong?

Reply:
Yes it does my bad I did not see an issue when i was testing it in my local game.

Reply:
I did see it before merging, but since its only a definition table it doesn't really matter :)

Reply:
Let me know if you need any help getting things fully updated for wrath, more than happy to help where I can. Not a professional with LUA but I can stumble my way through things.